vlib/v/checker/tests/import_mod_duplicate_err.out

```diff
@@ -0,0 +1,12 @@
+vlib/v/checker/tests/import_mod_duplicate_err.vv:3:10: error: cannot import `x.json2` as `json2` into a module with the same name
+    1 | module json2
+    2 | 
+    3 | import x.json2
```

Comment:
Hmm... I though you had separate messages for plain import vs import with alias?

Reply:
That's what I thought as well. The checker code didn't change since making the screenshots. Now I'm also getting the "wrong" error that was generated into the `err.out`'s.

The issue is here:
```v
if c.mod == ast_import.mod && ast_import.mod == ast_import.alias {
```

`ast_import.mod` is `x.json2` while it's alias `ast_import.alias` is `json2` when no other alias is assigned to it. Why ever it worked before... In any case, good to have your eyes on it. Thanks!